client/termdb/TermdbVocab.js

```diff
@@ -474,7 +474,7 @@ export class TermdbVocab extends Vocab {
 		if (filter) {
 			body.filter = getNormalRoot(filter)
 		}
-		return await dofetch3('/termdb', { body })
+		return await dofetch3('/termdb/getdescrstats', { body })
```

Comment:
wrong route name. no `get`

Reply:
okay thanks for pointing. It's fixed now.